src/lib/settings.js

```diff
@@ -25,10 +27,16 @@ export const DEFAULT_SETTINGS = {
     startingRow: 0,
     placeColumn: 0,
     roundsColumns: null,
+    nameColumns: null,
+    nameColumnHeaders: ['name', 'player', 'gracz', 'imię'],
+    nameCellExpression : '(?=^.*[A-Z][a-z]{3,})(?!.*([Kk][yy][uu]|[Dd][Aa][Nn]))',
```

Comment:
So I'd suggest adding _ignore-case_ flag and yes, remove `d|k|p|` letters from RegExp - somehow I missed the part with `{3,}` when parsing expression in my mind :)